src/jit/hwintrinsicxarch.cpp

```diff
@@ -545,10 +545,40 @@ bool Compiler::compSupportsHWIntrinsic(InstructionSet isa)
                                                     isFullyImplmentedISAClass(isa));
 }
 
-static bool isTypeSupportedForIntrinsic(var_types type)
+//------------------------------------------------------------------------
+// signatureTypeSupproted: platform support of hardware intrinsics
+//
+// Arguments:
+//    retType - return type
+//    sig     - intrinsic signature
+//    flags   - flags of the intrinsics
+//
+// Return Value:
+//    On 32-bit platforms, return true if the intrinsic does not use any 64-bit registers (r64)
+bool Compiler::signatureTypeSupproted(var_types retType, CORINFO_SIG_INFO* sig, HWIntrinsicFlag flags)
```

Comment:
These needs to have HWIntrinsic in the name somewhere.

Reply:
Will do.